pallets/vector/src/state.rs

```diff
@@ -118,19 +119,20 @@ impl VerifiedStep {
 pub struct VerifiedRotate {
 	pub verified_function_id: H256,
 	pub verified_input_hash: H256,
-	pub sync_committee_poseidon: U256,
+	pub sync_committee_hash: U256,
 }
 
+
 impl VerifiedRotate {
 	pub(crate) const fn new(
 		verified_function_id: H256,
 		verified_input_hash: H256,
-		sync_committee_poseidon: U256,
+		sync_committee_hash: U256,
 	) -> VerifiedRotate {
 		VerifiedRotate {
 			verified_function_id,
 			verified_input_hash,
-			sync_committee_poseidon,
+			sync_committee_hash,
 		}
 	}
 }
```

Comment:
Do you use `VerifiedRotate` or `VerifiedStep` anymore? If not, why did you update these functions?